src/Render.cc

```diff
@@ -16,11 +16,15 @@ using namespace snowcrash;
 namespace drafter {
 
     // JSON content-type regex
-    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json";
+    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json[[:blank:]]*(;.*|$)";
+    const char* const JSONSchemaRegex = "^[[:blank:]]*application/schema\\+json[[:blank:]]*(;.*|$)";
```

Comment:
OK, from the test case I don't see it, as there is exact string `application/schema+json` and then something more which is just copied exactly to the output. But that's OK, I just wanted to point it out to give it a second thought if doing regex is necessary.If you say so I believe you.